main.go

```diff
@@ -118,6 +122,18 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *streamKey == "" {
+		uuid, err := newUUID()
+		if err != nil {
+			fmt.Println("Can't generate rtmp key:", err)
+			os.Exit(1)
+		}
+
+		*streamKey = uuid
+	}
+
+	fmt.Println("RTMP stream key set to: ", *streamKey)
```

Comment:
You can remove the trailing space here. `fmt.Println` will concatenate strings with a space in-between automatically.

Reply:
Ah right!  I noticed that extra space and forgot to remove it.